FEM/rf_bc_new.cpp

```diff
@@ -513,14 +551,6 @@ std::ios::pos_type CBoundaryCondition::Read(std::ifstream* bc_file,
             in.clear();
         }
         //....................................................................
-        // aktive state of the bc is time controlled  WX
-        if (line_string.find("$TIME_CONTROLLED_ACTIVE") != std::string::npos)
```

Comment:
would you please output error message if this keyword is found in input file?

Reply:
Do you mean for the keyword of $TIME_CONTROLLED_ACTIVE?

For the new keyword, an error message for missing the keyword is unnecessary because the new keyword is optional.

Reply:
Yes I mean $TIME_CONTROLLED_ACTIVE, because this keyword is deleted after merging this PR. Those who using older versions and using this keyword will surely be confused.

Reply:
Added.

Reply:
thanks!